fvm/evm/stdlib/contract.cdc

```diff
@@ -7,6 +7,18 @@ contract EVM {
     access(all)
     event CadenceOwnedAccountCreated(addressBytes: [UInt8; 20])
 
+    /// FLOWTokenDeposit is emitted when FLOW tokens is bridged 
```

Comment:
I think we are using past tense for events. `FLOWTokensDeposited`.

Reply:
Update those to use past participle.